### DISCUSSION TITLE
How to troubleshoot the black screen issue on reTerminal | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#878] [Q&A]
# How to troubleshoot the black screen issue on reTerminal | Seeed Studio Wiki

reTerminal-FAQ

https://wiki.seeedstudio.com/reterminal_black_screen/

**Comment by DonEnar** [comment]
Hi

I have a black screen today, 2nd time i turned on my new reTerminal. First time i had no keyboard and no WiFi.
Today i had time to test my new toy.... so, first updates..... black screen after 15 minutes......
I found this page and tryed to connect to my PC (PWR and SYS are still on), how in step 2 described ... nothing showed up in my explorer.
What to do now? 
Kind regards S.Schlegel

**Comment by Colinb123** [comment]
Do you even test this stuff? This page is not helpful. At the top of the page, it tells you to install the original. At the bottom it has new files. The 64bit one doesn't work. The screen is locked and unresponsive. I was optimistic for this device, but so far standard Pi4 is much more reliable. I've rebuilt it 3 times now. I now know why you include a screwdriver. Seems to be working then black screen again. I can remote in but not always. And the instructions for flashing STM are wrong. There are 2 lines that need to be ##. Extremely disappointed, I was planning to buy hundreds of these for a project.

**Comment by jakobi** [comment]
For openocd to be able to use the gpio pins, I had to copy-edit openoct-code/interface/sysfsgpio-raspberrypi.cfg and add 512 to the gpio numbers.

I used the base from pinctrl, which can be found here: grep . /sys/class/gpio/gpiochip*/{base,label} | sort 

Openocd was from sf.net (follow the links on openocd.org), as I switched after the initial failure (likely not necessary?). Kernel is 6.6, raspios is the official one via rpi-imager: bookworm 20240315, kernel 6.6.

For sane chances of flashing, I used the distancer and two pogo pins from one of those 2.0 mm pogo programming clamps.

Final result: I went from pre 1.7 to 1.9 in one step, no flashing issues (other than openocd adding spurious debug output from the stm32), screen is working now.
